### ISSUE TITLE
Build a library of PDFs that are hard to parse

### ISSUE THREAD

**Issue by hammer** [refstudio/refstudio#111]
We should also look at the GROBID and AI2 repos to see if they have already built this collection...

**Comment by danvk** [comment]
Even just a random sample of PDFs (not necessarily hard to parse) would be helpful to let us say things like "we can parse metadata from 99% of academic paper PDFs".

**Comment by hammer** [comment]
@shauryr I've recently signed up for Azure and it would be fun to some day try the [Azure OpenAI Service On Your Data](https://techcommunity.microsoft.com/t5/ai-cognitive-services-blog/introducing-azure-openai-service-on-your-data-in-public-preview/ba-p/3847000) so my first thought is to put this library of PDFs into their object store. Let me know if you had a different plan.

**Comment by gjreda** [comment]
For now, we have been using the same set of ML-related PDFs I used when building the initial prototype of chat. You can find those PDFs [here](https://github.com/gjreda/scratch-pdf-bot/tree/main/papers).

**Comment by shauryr** [comment]
I was able to find the benchmark data used by grobid - [bioRxiv on zenodo](https://zenodo.org/record/3873702) and [PMC_sample_1943 s3 bucket](https://grobid.s3.amazonaws.com/PMC_sample_1943.zip).

They also benchmark on PLOS journal and eLife as well. More details here - https://grobid.readthedocs.io/en/latest/Benchmarking/

**Comment by shauryr** [comment]
I have uploaded these 25 pdfs on a huggingface [repo](https://huggingface.co/datasets/shaurya0512/bio_pdfs) because they are large in size for github.

**Comment by hammer** [comment]
Interesting PDF corpus https://arxiv.org/abs/2304.14953
